I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 207, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 390, in script
    $ __gl_test()
  File "renpy/common/00start.rpy", line 207, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 390, in script
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 390, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 342, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
OSError: Couldn't find file 'gui/window_icon.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "E:\renpy-8.1.3-sdk\renpy\bootstrap.py", line 275, in bootstrap
    renpy.main.main()
  File "E:\renpy-8.1.3-sdk\renpy\main.py", line 670, in main
    run(restart)
  File "E:\renpy-8.1.3-sdk\renpy\main.py", line 144, in run
    renpy.execution.run_context(True)
  File "E:\renpy-8.1.3-sdk\renpy\execution.py", line 955, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 207, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 390, in script
    $ __gl_test()
  File "/home/tom/ab/renpy-build-fix/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 444, in raise_
  File "renpy/common/00start.rpy", line 207, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 390, in script
    $ __gl_test()
  File "E:\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "E:\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00gltest.rpy", line 390, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 342, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
  File "E:\renpy-8.1.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "E:\renpy-8.1.3-sdk\renpy\display\core.py", line 3539, in interact
    self.start()
  File "E:\renpy-8.1.3-sdk\renpy\display\core.py", line 2409, in start
    self.post_init()
  File "E:\renpy-8.1.3-sdk\renpy\display\core.py", line 2503, in post_init
    self.set_icon()
  File "E:\renpy-8.1.3-sdk\renpy\display\core.py", line 2546, in set_icon
    with renpy.loader.load(icon, directory="images") as f:
  File "E:\renpy-8.1.3-sdk\renpy\loader.py", line 833, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'gui/window_icon.png'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
NovellaNASHA 1.0
Sun Oct 22 17:52:22 2023
